.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: python
+dist: trusty
+sudo: required
```

Comment:
Not sure about `sudo: required` (I guess this is the default anyways.)
`dist: trusty` is required because of [`cmake_minimum_required(VERSION 2.8.12)`](https://github.com/ipa-mdl/catkin/blob/compile-options/test/mock_resources/src/compile_options/set_compile_options/CMakeLists.txt#L1), which is needed for  `add_compile_options` in the tests. As an alternative `add_definitions` could be used though.